_episodes/02-image-basics.md

```diff
@@ -97,54 +60,214 @@ color, but that each pixel can have a different color from its neighbors.
-center, and y coordinate values will get larger as they go down instead of
-up, like this:
-
-![Image coordinate system](../fig/01-image-coordinates.png)
-
-This is called a *left-hand coordinate system*. If you hold your left hand
-in front of your face and point your thumb at the floor, your extended index
-finger will correspond to the x-axis while your thumb represents the y-axis.
-
-![Left-hand coordinate system](../fig/01-left-hand-coordinates.png)
-
-Until you have worked with images for a while, the most common mistake that
-you will make with coordinates is to forget that y coordinates get larger
-as they go down instead of up as in a normal Cartesian coordinate system.
-
-## Color model
-
-Digital images use some color model to create a broad range of colors from
-a small set of primary colors. Although there are several different color
-models that are used for images, the most commonly occurring one is the
-*RGB (Red, Green, Blue)* model.
-
-The RGB model is an *additive* color model, which means that the primary
-colors are mixed together to form other colors. In the RGB model, the
-primary colors are red, green, and blue -- thus the name of the model.
-Each primary color is often called a *channel*.
+## Working with Pixels
+As noted, in practice, real world images will typically be made up of a vast number of pixels, and each of these pixels will be  one of potentially millions of colors. While we will deal with pictures of such complexity shortly, lets start our exploration with just 15 pixels in a 5 X 3 matrix with 2 colors and work our way up to that complexity, but first the necessary imports.
+
+ ~~~
+"""
+ * Python libraries for learning and performing image processing.*
+"""
+import numpy as np
+import skimage.io
+import skimage.viewer
+import matplotlib.pyplot as plt
+import ipympl
+~~~
+{: .language-python}
+
+Now that we have our libraries loaded, we will run a Jupyter Magic Command that will insure our images display in our juptyer document with pixel information that will help us more efficiently run commands later in the session.
+ ~~~
+%matplotlib widget
+~~~
+{: .language-python}
+
+With that taken care of, let's load our image data from disk using the imread function from the skimage.io library and display it using the imshow function from the matplotlib library.
+ ~~~
+image = skimage.io.imread(fname="data/eight.tif")
+plt.imshow(image)
+~~~
+{: .language-python}
+![Image of 8](../fig/02-eight.png)
+You might be thinking, "That does look vaguely like an eight, and I see two colors but how can that be only 15 pixels". The display of the eight you see does use a lot more screen pixels to display our eight so large, but that doesn not mean there is information for all those screen pixels  in the file. All those extra pixels are a consequence of our viewer creating additional pixels through interpolation. It could have just displayed it as a tiny image using only 15 screen pixels if the viewer was written differently.
```

Comment:
```suggestion
You might be thinking, "That does look vaguely like an eight, and I see two colors but how can that be only 15 pixels". The display of the eight you see does use a lot more screen pixels to display our eight so large, but that does not mean there is information for all those screen pixels in the file. All those extra pixels are a consequence of our viewer creating additional pixels through interpolation. It could have just displayed it as a tiny image using only 15 screen pixels if the viewer was written differently.
```